### PR TITLE
combine playbooks

### PR DESCRIPTION
**Pull request by BerrieV1**

### CODE REVIEW

**Comment by BerrieV1** [comment]
combine playbooks

File: ansible/FLWSB-backend-ansible.yaml [hunk 1]
```diff
@@ -2,25 +2,31 @@
...
+      - nodered.iot-ap.be
   tasks:
 
 # RUN THIS PLAYBOOK: ansible-playbook FLWSB-backend-ansible.yaml -K
 # -K -> ask become password (root)
-
     - name: Update apt cache
       ansible.builtin.apt:
-        update_cache: true
+         update_cache: true
```
Comment (Jappie3) [inline ansible/FLWSB-backend-ansible.yaml:15]: one space too much I think

File: ansible/FLWSB-backend-ansible.yaml [hunk 2]
```diff
@@ -2,25 +2,31 @@
...
           - ca-certificates
           - curl
           - gnupg
           - lsb-release
           - pip
-      become: true
+          - nginx
+          - certbot
+          - python3-certbot-nginx
+        state: present
```
Comment (Jappie3) [inline ansible/FLWSB-backend-ansible.yaml:29]: add 'become: true'

**Review by Jappie3 — CHANGES_REQUESTED** [review]
looks good, 2 minor fixes needed

**Review by Jappie3 — APPROVED** [review]

**Comment by Jappie3** [comment]
add nginx & certbot to playbook

**Pull request merged**